.circleci/real_config.yml

```diff
@@ -1474,6 +1474,7 @@ jobs:
   lint-react:
     docker:
       - image: cimg/node:20.1.0
+    resource_class: large
```

Comment:
`large` seems large.  This run (https://app.circleci.com/pipelines/github/determined-ai/determined/47893/workflows/7712c52f-47e1-42fd-ae43-fee342918310/jobs/2048023/resources) peaked at 22% of the RAM on the system, and was mostly under 50% CPU utilization.  Given that a large is 4 CPUs and 8GB, I'd propose that a medium runner (2 CPUs and 4GB) would probably be more appropriate.  It appears that the test can at best use 2CPUs, and memory consumption is barely 2GB.  I think `small` might actually be perfect, but leaving some space and getting the likely runtime gain from 2 CPUs seems prudent.
```suggestion
    resource_class: medium
```

It seems that the default class is `medium`.  Looking at [the past 30 days](https://app.circleci.com/insights/github/determined-ai/determined/workflows/lint/jobs?branch=b1d8acd4-8701-4c76-8a87-7ff2320aa95f&reporting-window=last-30-days&job-name=lint-react) shows CPU utiliation averaging 60% and peaking at 100%, which should be ideal.  Memory usage averages ~20% and peaks at ~60% on the medium.  So based on that I don't think increasing the runner class will make a difference.

If we look at the performance on just `main`, this test has a 94% success rate for the past month.  I don't think this is the solution, but I'd like to spend some more time looking at failures to see what's actually failing.